script/vsts/platforms/templates/preparation.yml

```diff
@@ -37,23 +37,6 @@ steps:
     displayName: Setting globally used env variables
 
   # Windows Specific
-  - task: UsePythonVersion@0
-    inputs:
-      versionSpec: '3.8'
-    condition: eq(variables['Agent.OS'], 'Windows_NT')
```

Comment:
We should use Python 3 or 2 on all operating systems. This will fix #51 too.
```
  - task: UsePythonVersion@0
    inputs:
      versionSpec: '3.8'
```